bugshaker/src/main/java/com/github/stkent/bugshaker/ScreenshotUtil.java

```diff
@@ -0,0 +1,36 @@
+package com.github.stkent.bugshaker;
+
+import java.io.File;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+/**
+ * Created by aschen on 7/14/16.
+ */
```

Comment:
We should remove this comment. Maybe look up a way on AndroidStudio to prevent creating a comment when you create a new file.